assignment/impl/src/java/org/sakaiproject/assignment/impl/reminder/AssignmentDueReminderServiceImpl.java

```diff
@@ -111,10 +119,16 @@ public void execute(String opaqueContext) {
 
             // Do not send reminders if the site is unpublished or softly deleted
             if (site.isPublished() && !site.isSoftlyDeleted()) {
-                for (Member member : site.getMembers()) {
-                    if (assignmentService.canSubmit(assignment, member.getUserId()) && checkEmailPreference(member)) {
-                        sendEmailReminder(site, assignment, member);
+                SecurityAdvisor advisor = (u, f, r) -> { return SecurityAdvisor.SecurityAdvice.ALLOWED; };
```

Comment:
What permission is needed here? Can we narrow it instead of allowing everything?

Reply:
In my opinion we should improve the inner behaviour of canSubmit. I believe some cases (submitted, group submission...) will be missed following this SecurityAdvisor path.

Reply:
@bgarciaentornos I agree, but that is not the focus of this ticket. This is just about localisation so we should open another ticket about what you describe. In fact, this was not even working before as canSubmit was always returning false in the quartz context. The whole email actually needs moving into the email template service. My plan was to open another ticket for that, also.

Reply:
@adrianfish Specially for those reasons (canSubmit didn't work but that is not the focus of this) I think permission management should be treated on a different ticket, like you more or less say.